toot/cli/tui.py

```diff
@@ -24,12 +24,18 @@
     help=f"""Number of colors to use, one of {COLOR_OPTIONS}, defaults to 16 if
              using --color, and 1 if using --no-color."""
 )
+@click.option(
+    "-V", "--default-visibility",
+    callback=validate_visibility,
```

Comment:
You can avoid having a callback here:

```suggestion
    type=click.Choice(VISIBILITY_CHOICES),
```

Reply:
thanks, i changed that.